src/js/controller/controller.js

```diff
@@ -162,6 +163,14 @@ Object.assign(Controller.prototype, {
                 const type = streamType(duration, minDvrWindow);
                 model.setStreamType(type);
             });
+            if (Features.backgroundLoading) {
+                mediaModel.on('change:position', function (changedMediaModel, position) {
+                    if (!_programController.backgroundLoading
```

Comment:
You could remove the listener `mediaModel.off('change:position ,..` instead of checking `_programController.backgroundLoading`.